components/Layout.js

```diff
@@ -8,7 +8,7 @@ import MetaData from './MetaData'
 import { signOut } from 'next-auth/react'
 import PhaseBanner from './PhaseBanner'
 import Modal from 'react-modal'
-import { useEffect } from 'react'
+import { useEffect, useCallback } from 'react'
```

Comment:
Added useCallback, but never used.

Reply:
Was part of some previous testing. Missed this, thanks.